apps/src/templates/teacherDashboard/AddSectionDialog.jsx

```diff
@@ -25,9 +25,11 @@ import {
 // Navigates to the new section setup page if both params are non-null.
 const redirectToNewSectionPage = (participantType, loginType) => {
   if (!!participantType && !!loginType) {
-    navigateToHref(
-      `/sections/new?participantType=${participantType}&loginType=${loginType}`
-    );
+    const createSectionFromMyPl = participantType !== 'student';
```

Comment:
I'm curious ... would it be better to always redirect to the My PL page if the user created a PL section and otherwise direct to home? 

Not saying we should change what's here -- just thinking out loud!

Reply:
That's what these lines are effectively doing if I'm understanding your message properly! If it's a PL section, it redirects the user to the My PL page, otherwise it redirects home. I left the URL variable as `redirectToPage` in case we have other pages we want to redirect to in the future!

Reply:
ooo got it got it!